### ISSUE TITLE
Including combat_seq

### ISSUE THREAD

**Issue by cmatKhan** [mani2012/BatchQC#22]
Are there any plans to include combat_seq in BatchQC?

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7518324/

**Comment by WangKang-Leo** [comment]
Same question from me, it would be great if the BatchQC could integrate combat-seq

**Comment by wevanjohnson** [comment]
Yes! Hopefully very soon! On May 24, 2023, at 7:57 AM, WangKang-Leo ***@***.***> wrote:﻿
Same question from me, it would be great if the BatchQC could integrate combat-seq

**Comment by WangKang-Leo** [comment]
> Yes! Hopefully very soon! On May 24, 2023, at 7:57 AM, WangKang-Leo ***@***.***> wrote:﻿ Same question from me, it would be great if the BatchQC could integrate combat-seq —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

I am just wondering whether we could use count data (from RNA-seq) as input? Because combat wrapped in batchQC only receive normalized data matrix......
